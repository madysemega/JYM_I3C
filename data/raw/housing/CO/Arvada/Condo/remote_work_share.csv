predictions_remote_work_share
4.166526899337768
3.3716899967193603
3.7151208710670467
3.3112872815132137
2.940698776245117
2.638342385292053
3.102756069898605
2.822125027179718
2.5241957664489743
2.5183408379554746
2.3578286004066467
3.400887993574142
2.5605890941619873
2.157972079515457
2.6593906641006466
2.3124245357513424
2.38800865650177
2.5397639834880827
2.263792197704315
1.7288906288146972
1.556989848613739
1.6578465163707732
1.848872069120407
2.1324148488044736
2.125696711540222
1.711160806417465
1.9962522256374358
