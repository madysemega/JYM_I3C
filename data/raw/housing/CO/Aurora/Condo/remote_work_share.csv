predictions_remote_work_share
7.633496520519257
7.62232026219368
7.032914958000183
8.222759619951248
6.940736136436462
7.4040321862697605
6.850973854064941
7.4150219774246215
8.247977892160415
7.8422554647922516
8.785280219316483
8.173625520467759
7.580731490850448
7.92575602889061
6.739844599962234
7.522276387214661
6.977861467599869
8.601935902833938
7.311552003622055
7.522521243095398
9.000037661790849
7.177670458555221
8.485594985485077
6.710510865449906
7.618686035871506
7.619735149145127
7.242805418968201
