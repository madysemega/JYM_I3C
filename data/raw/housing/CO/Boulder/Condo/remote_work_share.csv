predictions_remote_work_share
15.691203837990763
17.10383320987225
19.083745499253276
16.69509342491627
17.272957866787912
16.36519653260708
14.333657477498056
16.68036151349545
15.584305031299593
14.592823221683505
12.437849920392038
15.233784289360049
15.245484496951105
16.079131776690485
16.092934034466744
17.448873127102853
17.543580042719842
16.590179945230485
15.398087142705918
14.524280891418458
16.01140444278717
14.275882602334024
14.211994737982751
14.545912898778917
15.725640274882318
17.196635368466378
16.40455187022686
