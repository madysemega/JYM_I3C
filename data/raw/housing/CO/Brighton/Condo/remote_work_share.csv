predictions_remote_work_share
10.890668935775755
5.137148843407631
9.289055060148238
10.819007638692854
5.310257248878479
9.031670672893524
11.324775577783583
9.990739027261734
9.022308279275894
9.5434499502182
4.546457178890705
5.12771598637104
7.8031363427639
5.254912276268005
11.658208476305006
11.550416772365569
11.697904794216155
7.081705291271209
11.06331124305725
8.272556709051132
7.369022769927978
6.186572093367576
10.951562036275863
7.145662395358086
12.4945734500885
11.995989314317702
11.852583550214767
