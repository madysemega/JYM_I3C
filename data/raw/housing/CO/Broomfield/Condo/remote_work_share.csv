predictions_remote_work_share
17.936834012866026
9.878764360547066
16.552901929020884
10.375892269015313
7.981048321127893
16.42635420203209
18.10321355104447
10.275154566168787
16.04887301564217
10.823715456724168
10.42590436577797
13.567507373690606
10.192857705950738
9.840540500879289
17.613489579558376
9.473359199762346
8.43532784819603
12.8668064647913
12.128297739624978
10.445813115835191
13.350406318306925
9.815303941965105
17.61861294448376
14.758419852852823
10.440135707855227
15.925055637359621
12.333624868988993
