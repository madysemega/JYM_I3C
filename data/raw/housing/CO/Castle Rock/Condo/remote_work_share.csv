predictions_remote_work_share
4.551303701400756
5.405395832061767
5.788341464996337
3.313045449256897
4.059137377738952
2.7754825377464294
3.835301132202148
3.802856788635254
4.278472881317138
3.1109674167633052
3.359421644210815
3.2812027168273925
4.451810355186462
4.401610865592956
4.6985569143295285
3.4405473995208737
3.4847372102737424
3.3859714174270628
2.937327613830566
3.330102834701538
3.771860780715942
4.00506516456604
3.7117592525482173
3.171920070648193
3.794378032684326
4.563721537590027
3.62697169303894
