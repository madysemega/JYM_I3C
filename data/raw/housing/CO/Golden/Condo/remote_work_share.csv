predictions_remote_work_share
7.652125031948089
10.31523435473442
11.52813367128372
10.628918073177337
9.850139324665069
11.616942205429076
9.269751009941102
10.04892674088478
11.434143471717833
8.562275468111038
9.02354938864708
12.1489638030529
9.593339624404907
10.629901311397552
9.308687243461609
11.042297230958937
11.964427195787428
11.836482093334196
10.154449095726013
9.027355749607086
11.242816375494002
9.829338920116424
10.982052996158599
9.48001035809517
9.097404098510742
12.53032012104988
10.205624186992646
